config.py

```diff
@@ -1049,9 +1049,9 @@
                'To turn it on, set use_ica=True.')
         logger.info(msg)
 
-if on_error not in ('continue', 'abort'):
-    msg = (f"on_error must be one of 'continue' or 'abort', but received "
-           f"{on_error}.")
+if on_error not in ('continue', 'abort', 'debug'):
+    msg = (f"on_error must be one of 'continue', 'debug' or 'abort', "
+           f"but received: {on_error}.")
```

Comment:
Whoops, did I accidentally undo your change here? Good thing you caught it!

Reply:
ok no worries